openslides/utils/access_permissions.py

```diff
@@ -1,46 +1,35 @@
-from django.dispatch import Signal
+from typing import Any, Dict, List, Optional, Union
 
-from .collection import Collection
-from .dispatch import SignalConnectMetaClass
+from django.db.models import Model
+from rest_framework.serializers import Serializer
 
+from .collection import Collection, CollectionElement
 
-class BaseAccessPermissions(object, metaclass=SignalConnectMetaClass):
```

Comment:
Why do you remove the metaclass here?  Maybe because the feature is not used any more ...?

Reply:
Yes. And I look forward of removing utils.dispatch anywhere. I think our new style, to explicit add objects to a list in app.raddy is match better.